src/utils.sh

```diff
@@ -36,6 +36,7 @@ declare -a EXIT_CODES=(0    # success
                       )
 
 CORRELATION_VECTOR=""
+DeviceUniqueID=""
```

Comment:
nit: should we also make this all caps? we might want to go through and make globals capitalized in the same format

Reply:
I think that is a great idea and I would like to add a different task to track that work..